templates/Vagrantfile

```diff
@@ -1,13 +1,25 @@
+VAGRANT_COMMAND = ARGV[0]
+
 Vagrant.configure(2) do |config|
   # Start bash as a non-login shell
   config.ssh.shell = "bash -c 'BASH_ENV=/etc/profile exec bash'"
-
-  config.vm.synced_folder "#{$user_vars['DREAMBOX_PROJECT_DIR']}",
-    "/home/#{$user_vars['DREAMBOX_USER_NAME']}/#{$user_vars['DREAMBOX_SITE_ROOT']}"
+  
+  $user_vars["DREAMBOX_UID"] = 31415
 
   # Run user_setup
   config.vm.provision 'shell',
     inline: '/bin/bash /usr/local/bin/user_setup',
     # Pass user_setup ENV variables to this script
     :env => $user_vars
+
+  if VAGRANT_COMMAND == "ssh"
```

Comment:
Interesting! I was moving toward dropping the user creation stuff, but this may have just changed my mind.